auth/src/main/java/com/firebase/ui/auth/ui/account_link/SaveCredentialsActivity.java

```diff
@@ -117,7 +118,8 @@ public void onConnected(@Nullable Bundle bundle) {
         if (mProfilePictureUri != null) {
             builder.setProfilePictureUri(Uri.parse(mProfilePictureUri));
         }
-        Auth.CredentialsApi.save(mCredentialsApiClient, builder.build()).setResultCallback(this);
+        mActivityHelper.getCredentialsApi()
+                .save(mCredentialsApiClient, builder.build()).setResultCallback(this);
```

Comment:
Nit: With chains of calls like this (where n > 2) I prefer to line break for each call:

```
mActivityHelper.getCredentialsApi()
    .save(mCredentialsApiClient, builder.build())
    .setResultCallback(this);
```